### PR TITLE
Remove outdated HTML versions

### PR DESCRIPTION
**Pull request by danielfett**
It's on the IETF website after all, and viewing main.md works just fine.

### CODE REVIEW

**Comment by Sakurann** [comment]
umm... many implementors of sd-jwt as a vc would not know ietf process and where to look for the ietf htmls. I would prefer keeping main.md as-is

**Comment by peppelinux** [comment]
@Sakurann the problem is that html is very outdated and we don't have a CD to keep it updated and, anyway, it would be an online replica. Having in the README a clear pointer to the IETF official page of SD-JWT I don't feel any worry for our implementers

**Comment by Sakurann** [comment]
ah ok. I am ok deleting [github.io](https://oauthstuff.github.io/draft-selective-disclosure-jwt/draft-fett-selective-disclosure-jwt-00.html) for an HTML version.
but why do we need to remove [IETF OAuth WG](https://datatracker.ietf.org/doc/draft-fett-oauth-selective-disclosure-jwt/)?

File: README.md [hunk 1]
```diff
@@ -5,10 +5,9 @@ documents that support selective disclosure of claim values.
 
 Written in markdown for the [mmark processor](https://github.com/mmarkdown/mmark).
 
-This is a GitHub repository for a draft specification in [IETF OAuth WG](https://datatracker.ietf.org/doc/draft-fett-oauth-selective-disclosure-jwt/)
-For the latest version of the IETF draft, please see https://datatracker.ietf.org/doc/html/draft-fett-selective-disclosure-jwt
+This is a GitHub repository for a draft specification in the IETF OAuth WG. For the latest published version of this IETF draft, please see https://datatracker.ietf.org/doc/html/draft-fett-selective-disclosure-jwt
```
Comment (peppelinux) [inline README.md:8]: hem.. probably @Sakurann is right, can we reenable the [IETF OAuth WG](https://datatracker.ietf.org/doc/draft-fett-oauth-selective-disclosure-jwt/) link?

as it is in this change the url is published quite raw

Comment (Sakurann) [inline README.md:8]:
```suggestion
This is a GitHub repository for a draft specification in the [IETF OAuth WG](https://datatracker.ietf.org/doc/draft-fett-oauth-selective-disclosure-jwt/). For the latest published version of this IETF draft, please see https://datatracker.ietf.org/doc/html/draft-fett-selective-disclosure-jwt
```

Comment (danielfett) [reply README.md]: You're right, I added the link again!

**Review by peppelinux — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]

**Pull request merged**